sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/BeamSqlPipelineOptions.java

```diff
@@ -30,6 +30,12 @@
 
   void setPlannerName(String className);
 
+  @Description("Default timezone for ZetaSQL analyzer; currently has no effect on CalciteSQL.")
```

Comment:
Can you add a note about which values are allowed here?

Reply:
Done.